doc/user_guide/index.rst

```diff
@@ -0,0 +1,33 @@
+.. toctree::
+     :maxdepth: 1
+     :hidden:
+
+.. toctree::
+    :caption: Getting started
+    :maxdepth: 1
+    :hidden:
+
+    installing.rst
+    loading_data.rst
+    distributed_big_data.rst
+
+.. toctree::
+    :caption: Usage
+    :maxdepth: 2
+    :hidden:
+
+    ../tutorials/index.rst
+    ../auto_examples/index.rst
```

Comment:
I believe this should be

```suggestion
    ../examples/index.rst
```

as per the [same file](https://raw.githubusercontent.com/pyxem/kikuchipy/develop/doc/user/index.rst) in kikuchipy's docs.

Reply:
It should be correct (I think) the difference is:

https://github.com/pyxem/kikuchipy/blob/b7d626ce52c9c049fefec59c0d103e5fc646434b/doc/conf.py#L338-L347

vs.

https://github.com/CSSFrancis/pyxem/blob/f1612457a3965a503265e46db499ec85052662b3/doc/conf.py#L178-L184

I just set that folder to `auto_examples`.  That is how `hyperspy` labels it and I think it helps to show that the folder shouldn't be edited.

Reply:
I agree that "auto" is a clear indication that the files are automatically generated. The URL will include "auto_examples" and not "examples", though (https://hyperspy--3238.org.readthedocs.build/en/3238/auto_examples/index.html). I find this more verbose, but it isn't a big issue.

Reply:
And then I don't know what causes the weird gallery.